pkg/generate/code/set_resource.go

```diff
@@ -1156,6 +1158,8 @@ func setResourceForContainer(
 	sourceVarName string,
 	// ShapeRef of the source struct field
 	sourceShapeRef *awssdkmodel.ShapeRef,
+	// currentPath is the absolute path to targetFieldName for nested fields
```

Comment:
It is actually the field path for the target *containing* field.

Reply:
Can you elaborate on the _containing_ part? It's not clear to me